app/src/main/res/layout/screen_enterotp.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="utf-8"?>
+<org.simple.clinic.enterotp.EnterOtpScreen xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  android:background="@color/window_background_auth"
+  tools:ignore="Overdraw">
+
+  <LinearLayout
+    android:id="@+id/enterotp_logo_container"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@color/color_primary"
+    android:orientation="vertical"
+    android:paddingBottom="@dimen/spacing_16"
+    android:paddingTop="@dimen/spacing_4">
+
+    <android.support.v7.widget.Toolbar
+      android:id="@+id/enterotp_toolbar"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:elevation="0dp"
+      app:navigationIcon="@drawable/ic_arrow_back_24dp"
+      tools:ignore="UnusedAttribute" />
```

Comment:
The navigation up icon is aligned with the card on Zeplin. Do you want to just use an `ImageButton` here? We also have a style: `Clinic.LoginBackButton`